src/controllers/vars_desktop.scss

```
@@ -57,6 +57,11 @@ $notification-width: 12.5rem !default;
   --current-left-panel-separator-width: #{$panel-separator-width};
   --current-left-panel-border-width: #{$border};
   --footer-panel-height: 12.5rem;
-  --footer-panel-height-lidarresult: 21.88rem;
+  --footer-panel-height-lidar: 21.88rem;
   --current-footer-panel-height: 0;
+
+  // LIDAR profile
+  --app-margin: #{$app-margin};
+  --lidarprofile-height: 21.88rem;
+  --profile-legend-width: 9.38rem;
```

```suggestion
  --current-footer-panel-height: 0;
  --app-margin: #{$app-margin};

  // LIDAR profile
  --footer-panel-height-lidar: 21.88rem;
  --lidarprofile-height: 21.88rem;
  --profile-legend-width: 9.38rem;
```
Better organisation :-)